service.subsmangler/resources/lib/__init__.py

```diff
@@ -0,0 +1 @@
+# Dummy file to make this directory a package.
```

Comment:
you probably know this, but python3 doesn't require those init placeholder files anymore

Reply:
Frankly speaking, I wasn't aware of this.
But as I look in Python 3.8 documentation [https://docs.python.org/3.8/tutorial/modules.html#packages](url) 
it still mentions this usage.
Let me look into it. Looks like more reading is needed on PEP420.

Reply:
Ok. I've removed __init__.py from resources/ and resources/lib